doc/rados/troubleshooting/memory-profiling.rst

```diff
@@ -60,112 +65,136 @@ in the interval. For instance::
      0.0   0.9%  97.7%      0.0  26.1% ReplicatedPG::do_op
      0.0   0.8%  98.5%      0.0   0.8% __gnu_cxx::new_allocator::allocate
 
-Refer to `Google Heap Profiler`_ for additional details.
+See `Google Heap Profiler`_ for additional details.
+
+After you have installed the heap profiler, start your cluster and begin using
```

Comment:
I think for most people the cluster will already be running.  I don't recall if the daemons need to restart after package installation.